docs/pywbemcli/commands.rst

```diff
@@ -1748,26 +1700,86 @@ the XML response from the WBEM server
+the interactive mode with a mock environment:
+
+.. code-block:: text
+
+    $ pywbemcli -n mock1
 
     pywbemcli> class enumerate --di --no
     CIM_Foo
     CIM_Foo_sub
     CIM_Foo_sub2
     CIM_Foo_sub_sub
 
-    Statistics: Time(Seconds); Times/lengths: (Avg/Min/Max), single value if all same.
-       Op    Exc     Op Time(S)  Operation
-      Cnt    Cnt    Avg/Min/Max
-    -----  -----  -------------  -------------------
-        1      0          0.000  EnumerateClassNames
+    pywbemcli> statistics show
+    Client statistics
+    Operation                    Count    Errors    Client Time    Server Time    Request Size    Response Size
+                                                           [ms]           [ms]             [B]              [B]
+    -------------------------  -------  --------  -------------  -------------  --------------  ---------------
+    compile_mof_file(ns=None)        1         0        149.862              0               0                0
+    CreateClass                      4         0          4.075              0               0                0
+    CreateInstance                  12         0          1.715              0               0                0
+    EnumerateClassNames              1         0          0.167              0               0                0
+    SetQualifier                    10         0          0.139              0               0                0
+
+The **Operation** column shows the name of the CIM-XML operation, plus the
+following additional entries:
+
+* compile_mof_file(ns=None)
+
+The **Count** column shows the number of operations executed.
+
+The **Errors** column shows the number of cases where the operation has
+resulted in an error at the level of the CIM-XML protocol. The occurrence of
+such errors is not necessarily a problem, depending on the logic in the
+pywbemcli client program.
+
+All time and size values in this report are average values across the number
+of operations executed, rounded to the precision shown.
+
+The **Client Time** column shows the total elapsed time the operation took from
+a perspective of the pywbemcli client program. This time includes network time
+and server time; since the measurement point in the client program is very
+close to the network interface, the time includes only a very small fraction of
+the time spent in code of the pywbemcli client program.
+
+The **Server Time** column shows the total elapsed time the operation took from
+a perspective of the WBEM server. It has the same meaning as the Server Time
+value shown in the server statistics. This time includes time spent in the
+CIM object manager code and time spent in its providers. The Server Time is
+obtained from the header field ``WBEMServerResponseTime`` in the CIM-XML
+response message; if the WBEM server does not support returning this field or
+has it disabled, the Server Time is shown as 0. That is why the Server Time
```

Comment:
The 0 is not created by pywbemcli: In case of the client statistics, it is created in pywbem. In case of the server statistics, it is created in the WBEM server (e.g. for class operations, Provider Time is 0).

**DISCUSSION**: Do we really want to change this?

Reply:
I agree with your option 1.

Reply:
Option 1 will need to be implemented in the pywbem statistics support. I have created issue https://github.com/pywbem/pywbem/issues/2603 for that. 
DONE here.